### PR TITLE
Update config.rst

### PR DESCRIPTION
**Pull request by AudriusButkevicius**

### CODE REVIEW

File: users/config.rst [hunk 1]
```diff
@@ -180,12 +180,15 @@ autoNormalize
 The following child elements may exist:
 
 device
-    These must have the ``id`` attribute and nothing else. Mentioned devices
-    are those that will be sharing the folder in question. Each mentioned
-    device must have a separate ``device`` element later in the file. It is
-    customary that the local device ID is included in all repositories.
+    These must have the ``id`` attribute and can have a ``introducedBy`` attribute,
```
Comment (ProactiveServices) [inline users/config.rst]: "...can have a..." should be "...can have an..."

File: users/config.rst [hunk 2]
```diff
@@ -180,12 +180,15 @@ autoNormalize
...
+    identifying the device that introduced us to share this folder with the given device.
+    If the original introducer unshares this folder with this device, our device will follow 
+    (subject to skipIntroductionRemovals being false on the introducer device) and unshare the folder.
```
Comment (ProactiveServices) [inline users/config.rst]: May be worth moving the parenthesised sentence to the end, as a fragment, to make this easier to read.

**Comment by calmh** [comment]
Minor language comments to address, otherwise good to merge as this functionality is merged now

**Comment by AudriusButkevicius** [comment]
Addressed.

**Comment by calmh** [comment]
@st-review merge it

specs/bep, users/config: Introducer removals

**Comment by st-review** [comment]
:ok_hand: Merged as a0f1a8d95e98aae1867b3fe516c98371a9a5a09b. Thanks, @AudriusButkevicius!

**Pull request closed without merging**